composer.json

```diff
@@ -25,6 +25,7 @@
         "consolidation/robo": "^1.4.12 || ^2",
         "dflydev/dot-access-data": "^1.1.0",
         "doctrine/annotations": "^1.10.0",
+        "drupal/config_split": "^1.7",
```

Comment:
Maybe also allow for Config Split v2 to be more forward compatible?

Reply:
oh that's a great call @lpeabody. i just grabbed a rule and didn't think much about it!